templates/flutter/lib/src/client_io.dart.twig

```diff
@@ -80,7 +80,7 @@ class ClientIO extends ClientBase with ClientMixin {
   @override
   String get endPoint => _endPoint;
 
-  Future<Directory> _getCookiePath() async {
+  Future<Directory> getCookiePath() async {
```

Comment:
do we really want to make this public?

Reply:
Well, due to recent change in Flutter, the plugins don't work in Flutter test, in order to run test successfully, I need to overwrite this method and can't do that without making it public

Reply:
We can discuss this in the other [thread](https://github.com/appwrite/sdk-generator/pull/607/files#r1092632600).